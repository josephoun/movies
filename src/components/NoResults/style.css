.no-results {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 100px;
}

.no-results p {
    font-size: 20px;
    font-weight: bold;
}
