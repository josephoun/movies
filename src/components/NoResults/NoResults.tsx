import React from "react";
import "./style.css";

const NoResults = () => {
    return (
        <div data-testid="no-match" className="no-results">
            <p>No matching results</p>
        </div>
    );
};

export default NoResults;
