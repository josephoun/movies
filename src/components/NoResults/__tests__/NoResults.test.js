import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom'
import NoResults from "../NoResults";

describe("NoResults component", () => {

    it("should match snapshot", () => {
        const { container } = render(
            <BrowserRouter>
                <NoResults />
            </BrowserRouter>,
        );
        expect(container).toMatchSnapshot();
    });
});
