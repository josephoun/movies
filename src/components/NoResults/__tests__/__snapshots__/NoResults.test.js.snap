// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component should match snapshot 1`] = `
<div>
  <div
    class="no-results"
    data-testid="no-match"
  >
    <p>
      No matching results
    </p>
  </div>
</div>
`;
